google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/main/java/com/google/cloud/spanner/jdbc/JdbcTypeConverter.java

```diff
@@ -0,0 +1,274 @@
+/*
```

Comment:
think this was again already added

Reply:
I based this PR on step 13 to avoid build errors. Once 13 is merged, this one should be rebased.